src/card/basic/card-basic.html

```diff
@@ -0,0 +1,9 @@
+<div ng-class="showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'" class="cardContainer">
```

Comment:
What is the purpose of this cardContainer class?  I don't see it used in css anywhere.